src/emoji/__tests__/data-test.js

```diff
@@ -67,6 +67,13 @@ describe('getFilteredEmojis', () => {
     ]);
   });
 
+  test('returns literal emoji', () => {
+    const list = getFilteredEmojis('🖤', {});
+    expect(list).toEqual([
+      { emoji_type: 'unicode', code: '1f5a4',  name: 'black_heart' },
+    ]);
+  });
```

Comment:
How does this behave in a case where we have several names for a given emoji? Per the jsdoc on `unicodeCodeByName`:
```
 * Many codes have multiple names.  For example: as of late 2018, there were
 * approximately 1600 names for 1000 codes, with about 400 codes having >=2
 * names, about 100 having >=3, up to as many as 6 names for the two
 * record-holders ('2620' and '1f5dd').
```

Probably a screenshot of the UI in that case would be helpful, as well as a test case demonstrating it.

Reply:
It shows the same emoji multiple times. I think this is somewhat reasonable, I added a test for it and a comment explaining my reasoning.

Reply:
> approximately 1600 names for 1000 codes, with about 400 codes having >=2

Not that it really matters, but 400 looks too low to me. Doesn't it have to be at least 600, and is that by the pigeonhole principle? 🤔

Reply:
The 1st code could have 601 names, and all of the rest could have one.  Pigeonhole principle here just says "1600 names and 1000 codes means _at least one code_ has more than one name."

Reply:
Ah, right, interesting—thanks, @alexmv!